src/lib/Behat/PageElement/Notification.php

```diff
@@ -49,4 +49,9 @@ public function closeAlert(): void
     {
         $this->context->findElement($this->fields['closeAlert'])->click();
     }
+
+    public function isAlertVisible(): bool
```

Comment:
There are no usages in this commit, is it intended? (I still believe that such method is useful, I'd rename it to `isVisible` and we can keep it).